ConsoleApp1/Program.cs

```diff
@@ -1,9 +1,18 @@
-﻿namespace ConsoleApp1
+﻿using VoteSystem;
+namespace ConsoleApp1
 {
-	internal class Program
-	{
-		static void Main(string[] args)
-		{
-		}
-	}
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            try
```

Comment:
this try-catch block is useless. Either remove it, or add right handler for all errors